packages/snippets/bin/extract_sample.dart

```diff
@@ -46,21 +47,29 @@ Future<void> main(List<String> argList) async {
     negatable: false,
     help: 'Prints help documentation for this command',
   );
+  parser.addFlag(
+    _kVerboseOption,
+    defaultsTo: false,
+    negatable: false,
+    help: 'Prints extra output diagnostics',
```

Comment:
nit: Some of the help texts here end with `.` and some don't. We should probably unify?

Reply:
Good point. Fixed.